src/EnergyPlus/InputProcessing/InputProcessor.cc

```diff
@@ -1905,9 +1913,11 @@ void InputProcessor::addRecordToOutputVariableStructure(std::string const &KeyVa
 
     auto const found = DataOutputs::OutputVariablesForSimulation.find(VarName);
     if (found == DataOutputs::OutputVariablesForSimulation.end()) {
-        std::unordered_map<std::string, DataOutputs::OutputReportingVariables,
+        std::unordered_map<std::string,
+                           DataOutputs::OutputReportingVariables,
                            UtilityRoutines::case_insensitive_hasher,
-                           UtilityRoutines::case_insensitive_comparator> data;
+                           UtilityRoutines::case_insensitive_comparator>
+            data;
```

Comment:
Bleck, something got funny with the indentation here.  I don't like the data word just hanging out down there.

Reply:
Blame the tyranny of clang-format for that.